addons/transifex/models/models.py

```diff
@@ -0,0 +1,86 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from configparser import ConfigParser
+from os.path import join as opj
+import os
+import werkzeug.urls
+
+import odoo
+from odoo import models
+
+class BaseModel(models.AbstractModel):
+    _inherit = 'base'
+
+    def get_field_translations(self, field_name, langs=None):
+        """ :return dict translations: [{lang: lang, source: val_en, value: val_lang, transifexUrl: transifex_url)] """
+        translations, context = super().get_field_translations(field_name, langs=langs)
+        self._get_transifex_url(translations)
+        return translations, context
+
+    def _get_transifex_url(self, translations):
+        """ Construct transifex URL based on the module on configuration """
+        self.ensure_one()
+        # .tx/config files contains the project reference
+        # using ini files like '[odoo-master.website_sale]'
+        external_id = self.get_external_id().get(self.id)
+        if not external_id:
+            return
+        module = external_id.split('.')[0]
+        if not module:
+            return
+
+        # e.g. 'https://www.transifex.com/odoo/'
+        base_url = self.env['ir.config_parameter'].sudo().get_param('transifex.project_url')
```

Comment:
@mart-e you know that this makes me happy right? Hearing it might go to V16 😉

Reply:
> Fallback on odoo if not present? (this was we can add the module in 16.0)

But `https://www.transifex.com/odoo/` is the `transifex.project_url`, I thought it was used to avoid hard-coding the url in the code. If the user changes `transifex.project_url`, we cannot fallback.
Or you just want me to hard code the `https://www.transifex.com/odoo/` here?